src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -139,25 +139,29 @@
   </Target>
 
   <Target Name="CreatePackNupkg">
+    <PropertyGroup>
+      <!-- Build from source can't use ILMerge. -->
+      <ILMergeSubpath Condition="'$(DotNetBuildFromSource)' != 'true'">ilmerge\</ILMergeSubpath>
+    </PropertyGroup>
     <ItemGroup Condition="'$(TargetFramework)' == '$(NETFXTargetFramework)' AND '$(IsBuildOnlyXPLATProjects)' != 'true'">
```

Comment:
Do you want the netfx assemblies when source building? 

I know they can be built, but not sure if you need them in the package.

Reply:
My understanding is we only use the source-built NuGet in the SDK we build for a certain platform, so there's no scenario where source-built netfx assemblies would be used. If someone wants the netfx NuGet bits to compile against, for example, when they add a package reference they'd get it from the NuGet gallery, not the source-built packages. No need to change this behavior AFAIK.

Reply:
I checked a finished source-build I have, and yep, the `net46` and `netstandard1.6` DLLs are both in the artifacts dir so it seems safe to add. Agreed this would be nice to have for consistency, but it does seem riskier and less likely to be used than the other change so I'll leave it for now.